FreeBSD 13 builds are failing due to KINFO_FILE_SIZE undefined

```
src/unix/kqueue.c: In function 'uv__fs_event':
src/unix/kqueue.c:496:25: error: 'KINFO_FILE_SIZE' undeclared (first use in this function); did you mean 'KINFO_PROC_SIZE'?
  496 |      kf.kf_structsize = KINFO_FILE_SIZE;
      |                         ^~~~~~~~~~~~~~~
      |                         KINFO_PROC_SIZE
src/unix/kqueue.c:496:25: note: each undeclared identifier is reported only once for each function it appears in
gmake[1]: *** [Makefile:2933: src/unix/libuv_la-kqueue.lo] Error 1
```

I'm cross-compiling to FreeBSD using FreeBSD 13 headers.

It looks like the most recent v1.x commit broke the build.

I'll have further details shortly after a bit more analysis and bisect.

--- a/src/unix/kqueue.c
+++ b/src/unix/kqueue.c
@@ -493,7 +493,13 @@ static void uv__fs_event(uv_loop_t* loop, uv__io_t* w, unsigned int fflags) {
 
   if (handle->event_watcher.fd != -1 &&
      (!uv__fstat(handle->event_watcher.fd, &statbuf) && !(statbuf.st_mode & S_IFDIR))) {
-     kf.kf_structsize = KINFO_FILE_SIZE;
+     /* we are purposely not using KINFO_FILE_SIZE here
+      * as it is not available on non intl archs
+      * and here it gives 1392 too on intel.
+      * anyway, the man page also mentions we can proceed
+      * this way.
+      */
+     kf.kf_structsize = sizeof(kf);
      if (fcntl(handle->event_watcher.fd, F_KINFO, &kf) == 0)
        path = uv__basename_r(kf.kf_path);
   }

